### ISSUE TITLE
Publish `src` folder on NPM

### ISSUE THREAD

**Issue by frostney** [planttheidea/moize#18]
Could you please publish the `src` folder on NPM as well? The compiled versions are problematic (in that it's resulting in errors) when trying to integrate this in React Native and defining custom Babel presets and plugins to use. Publishing the source folder would be an easy fix for this.

**Comment by planttheidea** [comment]
I mean if it comes down to it I can definitely do this, however I'd like to know a bit more about the problem before I do. I am only applying three exports that produce remaining code (two others relate to flow syntax and are stripped out):
* add-module-exports (plugin)
* latest (preset)
* stage-2 (preset)

Which of these are causing the issue? I haven't worked with React Native before so if there is a well-known issue with using one of these then I keep mindful of it going forward.

**Comment by planttheidea** [comment]
Alright I've done some research and I've seen several packages that said inclusion of a `.babelrc` breaks react native (github issues on several packages just asking to publish the package ignoring the `.babelrc`. While I didn't have a `.babelrc`, my babel config was in the `package.json` which may have been resulting in the same issue.

I have published a prepatch release where I extrapolated out the babel config to the `.babelrc` file and do not include the file on publish. Try upgrading to version `2.0.3-1` and let me know if that fixes the issue. If it does then I'll publish it as a proper patch release.
